config/default/filter.format.basic_html.yml

```diff
@@ -57,7 +57,7 @@ filters:
     status: true
     weight: -50
     settings:
-      allowed_html: '<b> <i> <em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id class> <h3 id class> <h4 id class> <h5 id class> <h6 id class> <img src alt data-entity-type data-entity-uuid data-align data-caption class height width> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button class alt title> <table class> <caption class> <tbody class> <thead> <tfoot> <th scope class> <td class> <tr> <a id href name target hreflang data-entity-substitution data-entity-type data-entity-uuid title class> <p class> <div class role id aria-*> <br> <span class> <i class> <drupal-media data-entity-type data-entity-uuid data-align data-caption data-* alt> <hr>'
+      allowed_html: '<b> <i class=""> <em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id class=""> <h3 id class=""> <h4 id class=""> <h5 id class=""> <h6 id class=""> <img src alt data-entity-type data-entity-uuid data-align data-caption height width class=""> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button alt title class=""> <caption class=""> <tbody class=""> <thead> <tfoot> <th scope class=""> <td class=""> <tr> <div role id aria-* class=""> <br> <hr> <table class=" hs-table--borderless"> <a id href name target hreflang data-entity-substitution data-entity-type data-entity-uuid title class=" hs-button hs-button--big hs-secondary-button hs-more-link hs-external-link hs-file-link hs-pdf-link hs-private-link"> <drupal-media data-entity-type data-entity-uuid data-align data-caption data-* alt data-entity-type data-entity-uuid data-view-mode data-align data-caption alt title> <p class=" hs-font-lead hs-font-splash hs-short-line-length hs-breakout-box hs-caption hs-credits"> <span class=" hs-emphasized-text">'
```

Comment:
The allowed attributes needs to be fixed.
Related to this issue: https://www.drupal.org/project/drupal/issues/2710427